packages/jahia/migrations/migrate-to-v8.yaml

```
@@ -94,3 +95,10 @@ actions:
         chown -h dd-agent: $custom_check_config
         systemctl restart datadog-agent
       user: root
+
+  setOrganizationNameEnvVar:
+    - cmd[${this}]: echo "${user.email}" | cut -d'@' -f1
+    - setGlobals:
+        organization_name: ${response.out}
+    - env.control.AddContainerEnvVars[${this}]:
+      vars: {"ORGANIZATION_NAME": "${globals.organization_name}"}
```

```suggestion

  setOrganizationNameEnvVar:
    - if(!globals.orgname):
      - script: |-
         return {"result": 0, "orgname": "${user.email}".split('@')[0]}
      - setGlobals:
         organization_name: ${response.orgname}
    - env.control.AddContainerEnvVars[${this}]:
      vars: {"ORGANIZATION_NAME": "${globals.organization_name}"}
```


This suggestion should only set the global once (and in a much faster way) instead of reset it on each nodes for each nodegroup.

---

Changes made